configure.ac

```diff
@@ -236,6 +236,9 @@ AC_CONFIG_FILES([tests/general/pio_tutil.F90:tests/general/util/pio_tutil.F90])
 
 AC_CONFIG_LINKS([tests/unit/input.nl:tests/unit/input.nl])
 
+# Define to solve intel compiler warning.
+AC_DEFINE([_GNU_SOURCE], [1], [solve strnlen declared implicitly warning on intel compiler])
```

Comment:
Better to put this up around line 159, where all the other AC_DEFINES happen. That way we can see them all together instead of having the scattered through the file.

Reply:
moved to line 159